python-lib/prologin/masternode/concoursquery.py

```diff
@@ -62,6 +62,8 @@
             stechec_match.status = %(match_status)s
           GROUP BY
             stechec_match.id, stechec_map.id
+          ORDER BY
+            stechec_match.priority DESC
```

Comment:
I'd make the ordering (per priority) deterministic to help with debugging and reproducibility and such. Add `stechec_match.id ASC` maybe?

Reply:
Done.